src/main/java/lotto/io/ConsoleLottoOutput.java

```diff
@@ -0,0 +1,70 @@
+package lotto.io;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import lotto.domain.LottoReceipt;
+import lotto.domain.Rank;
+
+public class ConsoleLottoOutput implements LottoOutput {
+    private final String MESSAGE_ASKING_MONEY = "구입금액을 입력해 주세요.";
+    private final String MESSAGE_RECEIPT_MESSAGE = "개를 구매했습니다.";
+    private final String MESSAGE_ASKING_WINNING_NUMBERS = "당첨 번호를 입력해 주세요.";
+    private final String MESSAGE_ASKING_BONUS_NUMBER = "보너스 번호를 입력해 주세요.";
+    private final String MESSAGE_RESULT = "당첨 통계\n---";
+
+    private final String FORMAT_PROFIT_PERCENTAGE = "총 수익률은 %.01f%%입니다.";
```

Comment:
```suggestion
    private final String FORMAT_PROFIT_PERCENTAGE = "총 수익률은 %,.01f%%입니다.";
```
이렇게 천 단위 쉼표까지 찍는 것이 요구 사항에 부합하는 것 같습니다.

Reply:
`ex. 100.0%, 51.5%, 1,000,000.0%`라는 문구를 지나쳤네요, 요구사항 맞게 작성하는 게 중요했는데 아쉽습니다 😢 꼼꼼하게 봐주셔서 고맙습니다!